File: php/applications/photo_asset_manager/src/hello.php
<?php
// Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
// SPDX-License-Identifier: Apache-2.0

use Aws\S3\S3Client;

function hello($data)
{
    $dataResult = var_export($data, true);

    $bucket = $data['Records'][0]['s3']['bucket']['name'];
    $key = $data['Records'][0]['s3']['object']['key'];

    $s3Client = new S3Client(['region' => 'us-west-2']);
    $contentType = $s3Client->headObject(['Bucket' => $bucket, 'Key' => $key])['ContentType'];

    return [
        'bucket' => $bucket,
        'key' => $key,
        'contentType' => $contentType,
        'dataResult' => $dataResult
    ];
}

File: python/example_code/medical-imaging/imaging_set_and_frames_workflow/hello.py
# Copyright Amazon.com, Inc. or its affiliates. All Rights Reserved.
# SPDX-License-Identifier: Apache-2.0

# snippet-start:[python.example_code.medical-imaging.Hello]
import logging
import boto3
from botocore.exceptions import ClientError

logger = logging.getLogger(__name__)


def hello_medical_imaging(medical_imaging_client):
    """
    Use the AWS SDK for Python (Boto3) to create an Amazon HealthImaging
    client and list the data stores in your account.
    This example uses the default settings specified in your shared credentials
    and config files.

    :param medical_imaging_client: A Boto3 Amazon HealthImaging Client object.
    """
    print("Hello, Amazon Health Imaging! Let's list some of your data stores:\n")
    try:
        paginator = medical_imaging_client.get_paginator("list_datastores")
        page_iterator = paginator.paginate()
        datastore_summaries = []
        for page in page_iterator:
            datastore_summaries.extend(page["datastoreSummaries"])
        print("\tData Stores:")
        for ds in datastore_summaries:
            print(f"\t\tDatastore: {ds['datastoreName']} ID {ds['datastoreId']}")
    except ClientError as err:
        logger.error(
            "Couldn't list data stores. Here's why: %s: %s",
            err.response["Error"]["Code"],
            err.response["Error"]["Message"],
        )
        raise


if __name__ == "__main__":
    hello_medical_imaging(boto3.client("medical-imaging"))
# snippet-end:[python.example_code.medical-imaging.Hello]
